### PR TITLE
Fix Event Threat Detection findings ingestion

### PR DESCRIPTION
**Pull request by daniel-cit**
What is in this PR:

- ETD Stackdriver parsing for "Bad IP" and "Anomalous IAM grant" updated for the new format of ETD findings in Stackdriver.
- ETD "Bad IP" SCC parsing updated for new finding format in SCC (nested fields instead of flat fields).
- Added SCC Parsing for "ETD Anomalous IAM" grant and "SSH brute force".
- Changed comparison of external users and users in the Policy binding in IAM revoke automation to be case insensitive.
- Updated description of Terraform variable "findings-project" to point the user to the project configured in the Google Cloud Console in "Security" > "Threat Detection"

Observations: 

- Stackdriver parsing for "SSH brute force" was not updated because findings in the new format were not found.
- New SCC Parsing for SSH brute force is based in the existing Stackdriver "SSH brute force". Detailed test were added to help future changes if needed.

### CODE REVIEW

File: providers/etd/anomalousiam/anomalousiam_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          "properties": {
+			   "sensitiveRoleGrant": {
+				    "members": ["user:john.doe@example.com", "user:jane.doe@example.com"]
+   				}
+  			},
+          "evidence": [{"sourceLogId": {"projectId": "onboarding-project"}}],
+		  "detectionCategory": {
+			"ruleName": "iam_anomalous_grant"
+		  }
+		},
```
Comment (tomscript) [inline providers/etd/anomalousiam/anomalousiam_test.go:48]: can u fix the formatting here

File: providers/etd/badip/badip_test.go [hunk 2]
```diff
@@ -30,10 +30,15 @@ func TestBadIP(t *testing.T) {
...
-					"detectionCategory_ruleName": "bad_ip",
-				  "properties_project_id": "test-project-15511551515",
-				  "properties_instanceDetails": "/projects/test-project-15511551515/zones/us-central1-a/instances/bad-ip-caller",
-				  "properties_location": "us-central1-a"
+					"detectionCategory": {
+						 "ruleName": "bad_ip"
+					},
+				    "properties": {
+					     "instanceDetails": "/projects/test-project-15511551515/zones/us-central1-a/instances/bad-ip-caller",
+                          "network": {
```
Comment (tomscript) [inline providers/etd/badip/badip_test.go:38]: formatting

File: providers/etd/sshbruteforce/sshbruteforce_test.go [hunk 3]
```diff
@@ -9,9 +9,42 @@ import (
...
+			  "state": "ACTIVE",
+			  "category": "Brute_force: SSH Brute Force",
+			  "externalUri": "https://console.cloud.google.com/home?project=onboarding-project",
+			  "sourceProperties": {
+					"detectionCategory": {
+						 "ruleName": "ssh_brute_force"
+					},
+        			"properties": {
+        				"project_id": "onboarding-project",
+        				"loginAttempts": [{
```
Comment (tomscript) [inline providers/etd/sshbruteforce/sshbruteforce_test.go:27]: formatting

**Comment by daniel-cit** [comment]
> meta comment - lets really aim for small PRs. so if you have to list 5 or so bullets, probably each bullet should be its own PR. this keeps reviews simple and quick

Hi Tom, good to see you again.

The last two bullets:

- Replacing `user == member` with `strings.EqualFold(user, member)` 
- Replacing  `description = "Project ID where security findings are sent to."` with `description = "Project ID where Event Threat Detection security findings are sent to by the Security Command Center. Configured in the Google Cloud Console in Security > Threat Detection."`

Are easy to move to two different pull requests, but I don't know which is the best strategy to slice  the changes in the parsing of the ETD findings.

We have three findings:

- "Bad IP"
- "Anomalous IAM grant" 
- "SSH brute force"

and two formats:

- Stackdriver logs
- SCC Notifications

and their parsing depends on code generated by the protobuf compiler reading from a single file `providers/etd/protos/etd.proto` which will be a cause of conflict in every PR.

Which would be better in this case?

- 3 pull requests by type of finding
- 2 pull requests by type of format 
- 6 pull requests by type of finding x type of format

**Pull request merged**